### ISSUE TITLE
Wrong Mathjax fonts rendering

### ISSUE THREAD

**Issue by dax44** [quarto-dev/quarto-cli#875]
Rendering my qmd file I have this warnings

```
/output/chtml/fonts/woff-v2/MathJax_Zero.woff (404: Not Found)
  /output/chtml/fonts/woff-v2/MathJax_Main-Regular.woff (404: Not Found)
  /output/chtml/fonts/woff-v2/MathJax_Math-Italic.woff (404: Not Found)
  /output/chtml/fonts/woff-v2/MathJax_Calligraphic-Regular.woff (404: Not Found)
  /output/chtml/fonts/woff-v2/MathJax_Size1-Regular.woff (404: Not Found)
  /output/chtml/fonts/woff-v2/MathJax_Size4-Regular.woff (404: Not Found)
  /output/chtml/fonts/woff-v2/MathJax_Size2-Regular.woff (404: Not Found)
  /output/chtml/fonts/woff-v2/MathJax_AMS-Regular.woff (404: Not Found)
  /output/chtml/fonts/woff-v2/MathJax_Size3-Regular.woff (404: Not Found)
```

and my latex formulas have strange fonts
<img width="434" alt="Zrzut ekranu 2022-05-14 o 14 39 59" src="https://user-images.githubusercontent.com/8738353/168426132-d3a818c9-e6fc-479b-ab7a-ff0881186fb3.png">

Any ideas?

**Comment by dax44** [comment]
And this is strange because in Visual model I've my formula in proper way
<img width="287" alt="Zrzut ekranu 2022-05-14 o 14 41 42" src="https://user-images.githubusercontent.com/8738353/168426200-91a7a0ac-764e-4f7c-9c72-1f676e233c1b.png">

**Comment by jjallaire** [comment]
Please provide a complete example that exhibits this behavior (i.e. the equation + all of document metadata) along w/ what platform you are running on and what version of quarto you are using.

**Comment by dax44** [comment]
My _quarto.yml file consists of

```
project:
  title: "Stat_mat_new_quarto"

format: 
  html:
    toc: true
    toc-title: Agenda
execute: 
  echo: true
  warning: false
self-contained: true
number-sections: true
editor: visual
editor_options: 
  chunk_output_type: console
```

and test.qmd file

```
---
title: "TEST"
---

::: hidden
$$
 \def\P{{\mathrm{P}}}
 \def\M{{\mathcal{P}}}
 \def\E{\mathrm{E}}
 \def\Var{\mathrm{Var}}
 \def\Cov{\mathrm{Cov}}
 \def\Cor{\mathrm{Cor}}
 \def\Tr{\mathrm{Tr}}
 \def\probit{\mathrm{probit}}
 \def\logit{\mathrm{logit}}
 \def\zb#1{{\buildrel #1 \over \longrightarrow}}
$$
:::

$$
    \dfrac{\sqrt{n}(F_n(t)-F(t))}{\sqrt{F(t)(1-F(t))}}\zb{d}N(0,1),\quad\text{gdy }n\to\infty.
$$ {#eq-slabazb}
```

Quarto version. 0.9.374, R 4.2.0, RStudio 2022.02.2 Build 485

**Comment by dax44** [comment]
When I change my html-math-method: katex all symbols are correct. Only my new definition won't work :(

**Comment by jjallaire** [comment]
The problem here appears to be `self-contained: true`. In theory Pandoc can do Mathjax self-contained but I can repro this with raw pandoc:

```bash
pandoc math.qmd --from markdown --output math.html --mathjax --standalone --self-contained
```

@dragonstyle Perhaps we should consider restoring our special handling of math in self-contained that was reverted here: https://github.com/quarto-dev/quarto-cli/commit/d69e9d33a6d2640fe8c5229be35b4fcbbf0bfb16. 

We can't just restore that commit as it does template patching, but we could have a partial for math that allows us to more elegantly do this?

**Comment by dax44** [comment]
@jjallaire thanks for your help. I'll be waiting for updates in self-contained settings.

**Comment by dragonstyle** [comment]
Ok this should be fixed as of this commit 48dff0551669bb55d07def3309ad5a678732ad1c. A fresh release is being built now and should be ready in a few minutes. Thanks for reporting this!
